commands/bench/bench.sh

```diff
@@ -75,12 +83,12 @@ main() {
     echo "Merging $GH_OWNER/$GH_OWNER_REPO#$GH_OWNER_BRANCH into $GH_CONTRIBUTOR_BRANCH"
     git remote add \
       github \
-      "https://token:${GITHUB_TOKEN}@github.com/${GH_OWNER}/${GH_OWNER_REPO}.git"
+      "https://token:${GITHUB_TOKEN}@github.com/${GH_OWNER}/${GH_OWNER_REPO}.git" || :
     git pull --no-edit github "$GH_OWNER_BRANCH"
-    git remote remove github
+    git remote remove github || :
   fi
 
-  set -x
+  # set -x
```

Comment:
Did you forget to put it back? :)

Reply:
no, i wanted to disable this, but enable then when/if it's needed )